auth/src/main/res/layout/register_email_layout.xml

```diff
@@ -1,90 +1,102 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Finish button shouldn't be inside Scrollview because it must be aligned above the keyboard.